r/R/dplyr.R

```diff
@@ -540,6 +542,31 @@ arrow_stringr_string_replace_function <- function(FUN, max_replacements) {
   }
 }
 
+arrow_r_string_split_function <- function(FUN, reverse = FALSE, max_splits = -1){
+  function(x, split, fixed = FALSE, perl = FALSE, useBytes = FALSE){
+    
+    assert_that(length(split) == 1)
```

Comment:
There's a handy function `assertthat::is.string()` that arrow imports that would be perfect here:
```suggestion
    assert_that(is.string(split))
```

Reply:
(it checks that the input is a character vector of length 1)